### PR TITLE
fix: git clone from bitbucket failed

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Git clone with "--depth" option will fail when the server is Bitbucket which version less than v0.6.4(This version is not guaranteed to be accurate, I tested v0.6.4 support "--depth", but v0.5.4.9 not support), so we will not use "--depth" when the server is Bitbucket.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/resolver/git/entrypoint.sh [hunk 1]
```diff
@@ -180,7 +187,7 @@ pull() {
         }
 
         echo "Fetch $SCM_REVISION from origin"
-        git fetch -v --depth=1 origin $SCM_REVISION
+        git fetch -v ${GIT_DEPTH_OPTION} origin $SCM_REVISION
```
Comment (cd1989) [inline build/resolver/git/entrypoint.sh]: Use `${GIT_DEPTH_OPTION:-}` to use default empty value if not variable set, otherwise

Comment (zhujian7) [reply build/resolver/git/entrypoint.sh]: PTAL

**Pull request merged**